opm/io/eclipse/ERft.hpp

```diff
@@ -63,18 +67,24 @@ class ERft : public EclFile
     bool hasArray(const std::string& arrayName, const std::string& wellName,
                   const RftDate& date) const;
 
+    bool hasArray(const std::string& arrayName, int reportInd) const;
+
+    int numberOfReports() { return numReports; }
+
 private:
-    std::map<int, std::pair<int,int>> arrIndexRange;
+    std::map<int, std::tuple<int,int>> arrIndexRange;
     int numReports;
     std::vector<float> timeList;
 
     std::set<std::string> wellList;
     std::set<RftDate> dateList;
     RftReportList rftReportList;
 
-    std::map<std::pair<std::string,RftDate>,int> reportIndex;  //  mapping report index to wellName and date (tupe)
+    std::map<std::tuple<std::string,RftDate>,int> reportIndex;  //  mapping report index to wellName and date (tupe)
```

Comment:
this member shadows with every single method taking reportIndex as a parameter. rename it, e.g. 'reportIndices', or change the parameter names in the methods.

Reply:
done